backend/samfundet/views.py

```diff
@@ -504,6 +504,7 @@ def list(self, request: Request) -> Response:
         Returns a list of all the recruitments for the specified gang.
         """
         recruitment_id = request.query_params.get('recruitment')
+        user_id = request.query_params.get('user_id')
```

Comment:
Du må nok håndtere dersom user_id er None også.
```suggestion
        user_id = request.query_params.get('user_id', request.user)
```

Reply:
Jeg ville tenkt den ikke kunne være det?

Reply:
Altså man må jo være logget inn for å sende request, så enten sender man inn en id, eller så blir user_id til brukeren som sender request? Og hvis det ikke er en bruker som sender request burde man ikke ha permission?

Reply:
Ah sant, vi er beskyttet av IsAuthenticated